app/views/projects/index.html.erb

```diff
@@ -1,4 +1,17 @@
-<h1>Projects</h1>
-<% @projects.each do |project| %>
-  <h2><%= project.name %></h2>
-<% end %>
+<div class="container">
+  <br>
+  <h1>Projects</h1>
+  <br>
+    <div class="row mb-3">
+    <% @projects.each do |project| %>
+      <div class="col-12 col-sm-6 col-lg-4 col-xl-3 d-flex align-items-stretch">
+        <div class="card mb-3 card-radius" >
+          <div class="card-body">
+            <h2><%= project.name %></h2>
+            <p>Names of team members to be displayed here</p>
+          </div>
+        </div>
+      </div>
+    <% end %>
+    </div>
+</div>
```

Comment:
This isn't a requirement for now, but it would be better to make this an `ul` with `li`s, as it's a list of things, and use CSS to style it to look how you want.

Reply:
It will be `ul` and `li` on the new version